src/c.mli

```diff
@@ -0,0 +1,45 @@
+open Stdune
+
+module Kind : sig
+  type t =
+    | C
+    | Cxx
+
+  val pp : t Fmt.t
+
+  val split_fn : string -> (string * t) option
```

Comment:
what about `split_extension`? To match `Filename.split_extension`

Reply:
done